wire-runtime-swift/src/test/swift/CodableTests.swift

```diff
@@ -52,7 +52,7 @@ extension CodableTests {
             opt_uint64: 4,
             opt_float: 5,
             opt_double: 6,
-            opt_bytes: Data(hexEncoded: "0123456"),
+            opt_bytes: Foundation.Data(hexEncoded: "0123456"),
```

Comment:
Would it make sense to instead make a few type aliases?

```swift
typealias Bytes = Data // Wire.Bytes
typealias UnknownFields = Data // Wire.UnknownFields
```

Basically to encourage semantic naming

Reply:
will update

Reply:
Doesn't that introduce the same problem, but with a message named `Bytes` (and/or `UnknownFields`)?

Reply:
It's for sure a similar problem. The UnknownFields PR is explicitly `Wire.UnknownFields`.

I would guess that `message Bytes` wouldn't happen due to there being a `bytes` type, but we could also be explicitly Wire.Bytes.

My thinking was that Data was doing double-duty for two very different use cases.
I'm also 100% OK with using `Foundation.Data` for `bytes`.